### ISSUE TITLE
New-tab command is broken

### ISSUE THREAD

**Issue by matheo** [Guake/guake#1542]
I just upgraded to `Ubuntu 19.04` and it comes with Guake 3.4.0:
```
Guake Terminal: 3.4.0
VTE: 0.56.1
VTE runtime: 0.56.1
Gtk: 3.24.8
```
and my custom `on-start-hook.sh` doesn't work properly anymore.

The new-tab command is broken
trying to run `guake -n "/path/to/the/desired/folder"` doesn't work
it always starts at the home folder.
Restarting Guake changes nothing

**Comment by matheo** [comment]
I've installed `Guake v3.5.1` from the source and it fixes the issue.

**Comment by ivoshm** [comment]
For the record - works even in released 3.5.0.
